client/my-sites/upgrades/checkout/free-trial-confirmation-box.js

```diff
@@ -0,0 +1,53 @@
+
+/**
+ * External dependencies
+ */
+var React = require( 'react' ),
+	classNames = require( 'classnames' );
+
+/**
+ * Internal dependencies
+ */
+var PayButton = require( './pay-button' ),
+	PaymentBox = require( './payment-box' ),
+	TermsOfService = require( './terms-of-service' );
+
+var FreeTrialConfirmationBox = React.createClass( {
+	content: function() {
+		return (
+			<form onSubmit={ this.props.onSubmit }>
+				<div className="payment-box-section">
+					<h6>{ this.translate( 'Your new site awaits' ) }</h6>
+
+					<span>
+					{
+						this.translate( 'Enjoy your free trial with no strings attached: your site will simply revert to the free plan when the period is over. We\'ll send you a reminder before the end of the trial, so look for our emails.' )
```

Comment:
This is a bit long. Let's remove the second sentence ("We'll send you a reminder before the end of the trial, so look for our emails.").